downloadmanager/src/main/java/cm/aptoide/pt/downloadmanager/DownloadAppFile.java

```diff
@@ -46,15 +46,16 @@ public String getPackageName() {
   }
 
   public int getFileType() {
-    return fileType.getType();
+    return fileType.
```

Comment:
I don't know if these codestyle changes are from the different os but if I put everything the same way it was before (getType on the same line) and apply the codestyle nothing changes

Reply:
yea, probably a misclick - done